minard/shifter_information.py

```diff
@@ -0,0 +1,103 @@
+import psycopg2
+from .db import engine
+from .views import app
+from wtforms import Form, StringField, validators
+from wtforms.fields.html5 import EmailField
+from wtforms.validators import ValidationError
+
+VALID_COUNTRIES = ['usa', 'canada', 'portugal', 'uk', 'germany']
+FORM_KEYS = ['firstname', 'lastname', 'expert', 'email', 'phone', 'country']
+
+class ShifterInfoForm(Form):
+
+    def validate_country(form, field):
+        if str(field.data).lower().strip() not in VALID_COUNTRIES:
+            raise ValidationError("Not a valid country")
+
+    def validate_phone(form, field):
+        if len(field.data) != 11:
+            raise ValidationError("Phone number wrong length")
+        if field.data and not str(field.data).isdigit():
+            raise ValidationError("Phone number must contain only digits")
+
+    def validate_expert(form, field):
+        found_expert = check_expert(field.data)
+        if not found_expert:
+            raise ValidationError("Expert could not be found.")
+
+    firstname = StringField('firstname', [validators.DataRequired()])
+    lastname = StringField('lastname', [validators.DataRequired()])
+    expert = StringField('expert', [validators.DataRequired(), validate_expert]) 
+    email = EmailField('email', [validators.Optional(), validators.Email()])
+    phone = StringField('phone', [validators.Optional(), validate_phone])
+    country = StringField('country', [validators.Optional(), validate_country])
```

Comment:
It looks like you are defining the validators twice. You can either create a separate function and then pass it in the list or if you name your function `validate_NAME` it will automatically be called.